### PR TITLE
Netty no flush

### PR DESCRIPTION
**Pull request by akurilov**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -57,7 +57,7 @@ ext {
 		grpc                     : "1.17.1",
 		guava                    : "28.1-jre",
 		log4j                    : "2.8.2",
-		mongooseBase             : "4.2.16",
+		mongooseBase             : "4.2.17",
 		mongooseStorageDriverPreempt: "4.2.21",
```
Comment (dadlex) [inline build.gradle]: Why do we rollback the preempt version?

Comment (dadlex) [reply build.gradle]: Should this be 4.2.23?

**Review by dadlex — APPROVED** [review]

File: build.gradle (hunk 1 above)
Comment (akurilov) [reply build.gradle]: rollback is for testing

File: src/test/robot/api/storage/pravega.robot [hunk 2]
```diff
@@ -36,7 +37,7 @@ Create Event Stream Test
 
 Create Byte Streams Test
     [Tags]  create_byte_streams
-    ${node_addr} =  Get Environment Variable  SERVICE_HOST  127.0.0.1
```
Comment (veronikaKochugova) [inline src/test/robot/api/storage/pravega.robot]: Why did you use the ip instead of SERVICE_HOST variable?

**Review by veronikaKochugova — APPROVED** [review]

File: src/test/robot/api/storage/pravega.robot (hunk 2 above)
Comment (dadlex) [reply src/test/robot/api/storage/pravega.robot]: I brought back the SERVICE_HOST variable, but set it to localhost as it's the ip we use to connect to pravega that now runs in docker

**Pull request merged**